uniffi/src/bindings/python/templates/ObjectTemplate.py

```diff
@@ -1,29 +1,25 @@
 class {{ obj.name()|class_name_py }}(object):
     # XXX TODO: support for multiple constructors...
     {%- for cons in obj.constructors() %}
-    def __init__(self, {% for arg in cons.arguments() %}{{ arg.name() }}{% if loop.last %}{% else %}, {% endif %}{% endfor %}):
-        {%- for arg in cons.arguments() %}
-        {{ arg.name()|coerce_py(arg.type_()) }}
-        {%- endfor %}
-        self._handle = _UniFFILib.{{ cons.ffi_func().name() }}(
-            {%- for arg in cons.arguments() %}
-            {{ arg.name()|lower_py(arg.type_()) }}{% if loop.last %}{% else %},{% endif %}
-            {%- endfor %}
-        )
+    def __init__(self, {% call py::arg_list_decl(cons.arguments()) -%}):
+        {%- call py::coerce_args_extra_indent(cons.arguments()) %}
+        self._handle = {% call py::to_rs_call(cons) %}
     {%- endfor %}
 
     # XXX TODO: destructors or equivalent.
 
-    {%- for meth in obj.methods() %}
-    def {{ meth.name()|fn_name_py }}(self, {% for arg in meth.arguments() %}{{ arg.name() }}{% if loop.last %}{% else %}, {% endif %}{% endfor %}):
-        {%- for arg in meth.arguments() %}
-        {{ arg.name()|coerce_py(arg.type_()) }}
-        {%- endfor %}
-        _retval = _UniFFILib.{{ meth.ffi_func().name() }}(
-            self._handle,
-            {%- for arg in meth.arguments() %}
-            {{ arg.name()|lower_py(arg.type_()) }}{% if loop.last %}{% else %},{% endif %}
-            {%- endfor %}
-        )
-        return {% match meth.return_type() %}{% when Some with (return_type) %}{{ "_retval"|lift_py(return_type) }}{% else %}None{% endmatch %}
-    {%- endfor %}
\ No newline at end of file
+    {% for meth in obj.methods() -%}
+    {%- match meth.return_type() -%}
+
+    {%- when Some with (return_type) -%}
```

Comment:
I'm wondering if there is a clean way to merge those 🤔 The difference seems to be the `_retval`. In Rust `_retval` can be `()` and that brings functions with a return_type and those without one closer. Not sure if that helps, or if the same concept exists in the bindings thou 😅  And I'd probably err on the side of consistency.. Anywhoo just dumping my random thoughts on an already great work 😆